span_test.go

```diff
@@ -573,6 +573,32 @@ func TestCompressSpanSameKind(t *testing.T) {
 			currentTime = currentTime.Add(span.Duration)
 			span.End()
 		}
+		// Span is compressable and the next span is the same kind, but cannot be compressed
+		// since the service target fields are different (inferred by the db instance).
```

Comment:
Would we lose any coverage by explicitly setting the service target? I think it would make the test a bit easier to understand. We already test the DB -> service target inference elsewhere.

Reply:
Good point! I don't think we'll lose anything. Updated :+1: